tools/heir-opt.cpp

```diff
@@ -452,6 +452,12 @@ void mlirToOpenFheBgvPipelineBuilder(
   auto secretToBgvOpts = SecretToBGVOptions{};
   secretToBgvOpts.polyModDegree = options.ciphertextDegree;
   pm.addPass(createSecretToBGV(secretToBgvOpts));
+}
+
+void mlirToOpenFheBgvPipelineBuilder(OpPassManager &pm,
+                                     const MlirToBgvPipelineOptions &options) {
+  // lower to BGV
+  mlirToBgvPipelineBuilder(pm, options);
```

Comment:
we can re-factor later once we have a ckks pass too, but maybe we should just have two levels of pass pipelines (1) mlir-to-bgv, mlir-to-ckks, mlir-to-cggi, etc, and then (2) something like lower-to-openfhe that bundles all the *-to-openfhe passes.